src/core/globalstate.ts

```diff
@@ -2,7 +2,11 @@ import {IDerivation} from "./derivation";
 import {Reaction} from "./reaction";
 import {IObservable} from "./observable";
 
-declare const global: any;
+declare let global: any;
+if (typeof global === "undefined" && typeof window !== "undefined")
+	global = window;
+
+
```

Comment:
@philcockfield, I beleave, with this implementation you'll get `Uncaught ReferenceError` in strict mode